docs/Text.md

```diff
@@ -0,0 +1,185 @@
+# Adding Text
+
+There are several ways in fpdf to add text to a PDF document, each of which comes with its own special features and its own set of advantages and disadvantages. You will need to pick the right one for your specific task.
+
+| method | lines | markdown | HTML | positioning | details |
```

Comment:
Great idea to provide this comparison table! Good job

"markdown" -> "markdown support" ?

Also, the meaning ot the "positioning" column doesn't seem very clear to me... "control text alignement" instead maybe?

Reply:
"markdown support" and "HTML support" will make it more explicit.
positioning is not related to alignment. "Accepts new end position" explains it better.